src/containers/orgs/OrgContainer.js

```diff
@@ -5,28 +5,148 @@
+  Card,
+  CardSegment,
+  Colors,
+  Input,
+  Spinner,
+} from 'lattice-ui-kit';
 import { connect } from 'react-redux';
+import { Route, Switch } from 'react-router';
+import { NavLink } from 'react-router-dom';
 import { bindActionCreators } from 'redux';
 import { RequestStates } from 'redux-reqseq';
 import type { RequestSequence, RequestState } from 'redux-reqseq';
 
-const Title = styled.h1`
+import * as ReduxActions from '../../core/redux/ReduxActions';
+import * as Routes from '../../core/router/Routes';
+
+const { NEUTRALS } = Colors;
+const { GET_ORGANIZATION } = OrganizationsApiActions;
+
+const OrgTitle = styled.h1`
   font-size: 28px;
   font-weight: normal;
-  margin: 20px 0 40px 0;
+  margin: 20px 0 0 0;
+  padding: 0;
+`;
+
+const OrgDescription = styled.h3`
+  font-size: 20px;
+  font-weight: normal;
+  margin: 0;
   padding: 0;
 `;
 
+const Grid = styled.div`
+  display: grid;
+  grid-gap: 30px;
+  grid-template-columns: 1fr 1fr;
+`;
+
+const GridItem = styled.div`
+  word-break: break-word;
+`;
+
+const SectionTitle = styled.h2`
+  font-size: 22px;
+  font-weight: 600;
+  grid-column: 1 / span 2;
+  margin: 0;
+`;
+
+const SectionSubTitle = styled.h2`
+  color: ${NEUTRALS[1]};
+  font-size: 16px;
+  font-weight: normal;
+  grid-column: 1 / span 2;
+  margin: 15px 0 30px 0;
+`;
+
+const ItemTitle = styled.h4`
+  color: ${NEUTRALS[1]};
+  font-size: 14px;
+`;
+
+const Divider = styled.div`
+  background-color: ${NEUTRALS[4]};
+  height: 1px;
+  margin: 48px 0;
+`;
+
+const Tabs = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  margin: 30px 0 50px 0;
+`;
+
+const AddNewWrapper = styled.div`
+  display: grid;
+  grid-gap: 5px;
+  grid-template-columns: 1fr auto;
+`;
+
+const AddButton = styled(Button)`
+  border-color: ${NEUTRALS[4]};
+  border-width: 1px;
+`;
+
+const ORG_NAV_LINK_ACTIVE :string = 'org-nav-link-active';
+
+const OrgNavLink = styled(NavLink).attrs({
+  activeClassName: ORG_NAV_LINK_ACTIVE
+})`
+  align-items: center;
+  border-bottom: 2px solid transparent;
+  color: ${NEUTRALS[1]};
+  font-size: 18px;
+  font-weight: 600;
+  line-height: 70px;
+  margin-right: 40px;
+  outline: none;
+  text-decoration: none;
+
+  &:focus {
+    text-decoration: none;
+  }
+
+  &:hover {
+    color: ${NEUTRALS[0]};
+    cursor: pointer;
+    outline: none;
+    text-decoration: none;
+  }
+
+  &.${ORG_NAV_LINK_ACTIVE} {
+    border-bottom: 2px solid #674fef;
+    color: #674fef;
+  }
+`;
+
+const ROLES_SUB_TITLE = `
+You will be able to use the Roles below to manage permissions on Entity Sets that you own.
+`;
+
+const MEMBERS_SUB_TITLE = `
+Click on a member to view their roles. To add members to this organization, search for users in the system.
+`;
+
 type Props = {
   actions :{
     getOrganization :RequestSequence;
+    resetRequestState :(actionType :string) => void;
   };
-  getOrgRequestState :RequestState;
   org :Map;
   orgId :UUID;
+  requestStates :{
+    [typeof GET_ORGANIZATION] :RequestState;
```

Comment:
why are you storing your `requestState` as `typeof GET_ORGANIZATION`?

Reply:
I'm trying a new thing here. I got tired of thinking of a variable name for the request state, and I wanted to use the action as the identifier, so this happened.

```
requestStates :{
  [typeof GET_ORGANIZATION] :RequestState;
};
```

^ this is for flow. What it means is that `requestStates` is an object with `typeof GET_ORGANIZATION` being a key in that object. If you look at how `GET_ORGANIZATION` is defined, you'll see:

```
const GET_ORGANIZATION :'GET_ORGANIZATION' = 'GET_ORGANIZATION';
```

The type of `GET_ORGANIZATION` is literally the string `GET_ORGANIZATION`, and that translates to flow resolving `typeof GET_ORGANIZATION` to the string literal `GET_ORGANIZATION`. In other words, with this, I don't have to think of and define a variable name like `getOrgRequestState`, and instead, I can reuse the existing `GET_ORGANIZATION` variable already defined and just do this:

```
// in mapStateToProps
return {
  requestStates: {
    [GET_ORGANIZATION]: state.getIn(['orgs', GET_ORGANIZATION, 'requestState']),
  },
};

// in the component
const { requestStates } = this.props;
... requestStates[GET_ORGANIZATION] ...
```

In more visual terms, this is what flow sees:

![Screen Shot 2019-07-30 at 3 32 12 PM](https://user-images.githubusercontent.com/440299/62170144-3fd6be80-b2df-11e9-8b71-a7c0d5e277a9.png)

Reply:
oh that's super cool! i hadn't thought much about how the `type of` the action is just the string literal of the action's name. seems like a great method to use going forward for naming request states!